lib/widgets/inbox.dart

```diff
@@ -340,6 +356,7 @@ class _DmItem extends StatelessWidget {
     return StickyHeaderItem(
       header: _AllDmsHeaderItem(
         count: allDmsCount,
+        isMentioned: isMentioned,
```

Comment:
This seems wrong — it's using the is-mentioned flag for the individual DM conversation, but passing it to use for the all-DMs header.

Hmm but I can't actually trigger any symptom from that bug.

… Aha. This header has no effect! That's a pre-existing latent bug in this file. What's happening is that the items of the list view are the entire sections — all DMs, then each stream — and not the individual conversations, so the `StickyHeaderItem` widgets on the individual conversations never get consulted.

I'll send a quick PR removing those `StickyHeaderItem`s, to avoid causing us similar confusion in the future. Then you can rebase atop that PR, and I think it will slightly simplify some of this code.

Reply:
#510